mobile/components/Inputs/CheckboxInput.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { useState } from "react";
+import { Checkbox, Label, XStack, CheckboxProps, CheckedState } from "tamagui";
+import { Icon } from "../Icon";
+import { Typography } from "../Typography";
+
+interface CheckboxInputProps extends CheckboxProps {
+  id: string;
+  label: string;
+}
+
+const CheckboxInput: React.FC<CheckboxInputProps> = ({ label, id, ...rest }) => {
+  const [isChecked, setIsChecked] = useState<CheckedState>(false);
```

Comment:
Here, this should come from outside, as i would like to have the posibility to set a default value. Do i have a "checked" prop and 'onChange' for this?